script.js

```diff
@@ -1,4 +1,210 @@
+var draws = 0;
+var playerWins = 0;
+var computerWins = 0;
+var timesPlayed = 0;
+var previousWinner = '';
+
+var mode = 'waiting for playerName';
+var playerName = '';
+
+// generate a random number between 1 and 3
+var randomNumber = function () {
+  return (Math.floor(Math.random() * 3)) + 1;
+};
+
+// game block
+var playGame = function (playerGuess) {
+  var computerChoice = '';
+  var computerNumber = randomNumber();
+  var message = '';
+
+  if (computerNumber == 1) {
+    computerChoice = 'scissors';
+  } else if (computerNumber == 2) {
+    computerChoice = 'paper';
+  } else {
+    computerChoice = 'stone';
+  }
+
+  // if both player and computer choose the same
+  if (playerGuess == computerChoice) {
+    draws += 1;
+    message = `${playerName}, it's a DRAW! The previous winner ${previousWinner} wins!<br><br>Your choice: ${playerGuess}<br>Computer's choice: ${computerChoice}<br>Player wins: ${playerWins}
+    <br>Computer wins: ${computerWins}<br>Draws: ${draws}<br>You have won ${playerWins}/${timesPlayed} times`;
+
+  // if player wins
+  } else if ((playerGuess == 'scissors' && computerChoice == 'paper')
+  || (playerGuess == 'paper' && computerChoice == 'stone')
+  || (playerGuess == 'stone' && computerChoice == 'scissors')) {
+    playerWins += 1;
+    previousWinner = playerName;
+    console.log(previousWinner);
+    message = `${playerName}, you WIN!<br><br>Your choice: ${playerGuess}<br>Computer's choice: ${computerChoice}<br>Player wins: ${playerWins}
+      <br>Computer wins: ${computerWins}<br>Draws: ${draws}<br>You have won ${playerWins}/${timesPlayed} times`;
+
+  // if computer wins
+  } else {
+    computerWins += 1;
+    previousWinner = 'computer';
+    console.log(previousWinner);
+    message = `${playerName}, you LOSE!<br><br>Your choice: ${playerGuess}<br>Computer's choice: ${computerChoice}<br>Player wins: ${playerWins}
+      <br>Computer wins: ${computerWins}<br>Draws: ${draws}<br>You have won ${playerWins}/${timesPlayed} times`;
+  }
+
+  return message;
+};
+// end of game block
+
+// reverse mode game block
+var reverseMode = function (playerGuess) {
+  var computerChoice = '';
+  var computerNumber = randomNumber();
+  var message = '';
+
+  if (computerNumber == 1) {
+    computerChoice = 'scissors';
+  } else if (computerNumber == 2) {
+    computerChoice = 'paper';
+  } else {
+    computerChoice = 'stone';
+  }
+
+  // if both player and computer choose the same
+  if (playerGuess == computerChoice) {
+    draws += 1;
+    message = `${playerName}, it's a DRAW!<br><br>Your choice: ${playerGuess}<br>Computer's choice: ${computerChoice}<br>Player wins: ${playerWins}
+    <br>Computer wins: ${computerWins}<br>Draws: ${draws}<br>You have won ${playerWins}/${timesPlayed} times`;
```

Comment:
This code is also duplicated. Consider using conditionals with boolean operations to combine code between reverse and normal game modes. E.g. if game mode is "reverse", then what used to be player wins is now computer wins and vice versa.